backend/src/main.ts

```diff
@@ -1,6 +1,6 @@
-import { Application } from "@reboot-dev/reboot";
+import { Application, ExternalContext } from "@reboot-dev/reboot";
 
-const initialize = async (context) => {};
+const initialize = async (context: ExternalContext) => {};
```

Comment:
`context` is unused.

Reply:
Yep, this can stay as it is imho and acts as a good entry point for this template, imho. :-)

Reply:
Hmm, I'm not sure how I feel about shipping an example that fails to pass lint check. If you want to keep this as-is, I'd suggest turning off linting for this line only with a comment explaining why.

Reply:
If this were an example, I would absolutely see it as you do.
As this is a template and not an example I personally have no problems with failing lint checks. Turning the linting off for this line is not a good idea I guess. If the templates gets filled with live, this switch could stay on accidentally.